ui/src/translations/en.json

```
@@ -51,6 +51,7 @@
   "message": "Message",
   "active_at": "Active Since",
   "cluster_up_and_running": "Everything is up and running",
+  "prometheus_unavailable": "Prometheus unavailable",
```

To be consistent with the French translation:

```suggestion
  "prometheus_unavailable": "Prometheus is unavailable",
```